terraform-dev/govgraphsearch.tf

```diff
@@ -187,18 +187,6 @@ data "google_secret_manager_secret_version" "iap_oauth_client_secret" {
   secret = "iap-oauth-client-secret"
 }
 
-data "google_secret_manager_secret_version" "sso_oauth_client_id" {
-  secret = "OAUTH_ID"
-}
-
-data "google_secret_manager_secret_version" "sso_oauth_client_secret" {
-  secret = "OAUTH_SECRET"
-}
-
-data "google_secret_manager_secret_version" "cookie-session-signature" {
-  secret = "cookie-session-signature"
-}
-
```

Comment:
It's possible we also need to keep this. The [deployment](https://github.com/alphagov/govuk-knowledge-graph-gcp/blob/ca488947c7e5ba3a94e0748ceaa9222c2def5583/terraform-dev/govgraphsearch.tf#L335) seems to use this value (and it is populated). I don't understand exactly why or how it's being used but might be related to use of this [package](https://www.npmjs.com/package/cookie-session)? Not sure.